samples/deploy/crd/v1/ApisixRoute.yaml

```diff
@@ -521,6 +521,8 @@ spec:
                       match:
                         type: object
                         properties:
+                          host:
+                            type: string
```

Comment:
please move this field to v2 api version section

Reply:
Done